### PR TITLE
(PA-3460) Add puppet7 cask

### PR DESCRIPTION
**Pull request by GabrielNagy**
Will have to run the rake task again when the rolling repo updates to puppet7.

### CODE REVIEW

**Comment by GabrielNagy** [comment]
```sh-session
> brew cask install GabrielNagy/puppet/puppet-agent-7
==> Tapping gabrielnagy/puppet
Cloning into '/usr/local/Homebrew/Library/Taps/gabrielnagy/homebrew-puppet'...
remote: Enumerating objects: 17, done.
remote: Counting objects: 100% (17/17), done.
remote: Compressing objects: 100% (14/14), done.
remote: Total 1307 (delta 5), reused 11 (delta 3), pack-reused 1290
Receiving objects: 100% (1307/1307), 1.18 MiB | 21.12 MiB/s, done.
Resolving deltas: 100% (696/696), done.
Tapped 9 casks and 2 formulae (51 files, 1.3MB).
==> Downloading https://downloads.puppet.com/mac/puppet7/10.15/x86_64/puppet-agent-7.0.0-1.osx10.15.dmg
######################################################################## 100.0%
==> Verifying SHA-256 checksum for Cask 'puppet-agent-7'.
==> Installing Cask puppet-agent-7
==> Running installer for puppet-agent-7; your password may be necessary.
==> Package installers may write to any location; options such as --appdir are ignored.
installer: Package name is puppet-agent
installer: Installing at base path /
installer: The install was successful.
🍺  puppet-agent-7 was successfully installed!
> puppet --version
7.0.0
```

**Review by Dorin-Pleava — APPROVED** [review]

**Review by MikaelSmith — CHANGES_REQUESTED** [review]
Can you add puppet-agent-7 to the automated install tests? https://github.com/puppetlabs/homebrew-puppet/blob/master/.travis.yml#L21-L23

**Comment by GabrielNagy** [comment]
nice catch! added ✔️

**Review by MikaelSmith — APPROVED** [review]

**Comment by GabrielNagy** [comment]
Looks like some cells are failing, I can work on updating the travis configuration.

File: .travis.yml [hunk 1]
```diff
@@ -21,6 +21,7 @@ env:
     - CASK=puppet-agent
```
Comment (MikaelSmith) [inline .travis.yml:21]: Our list of test platforms at the top seems to be getting old. May need to update it.

**Review by MikaelSmith — APPROVED** [review]
I think there's a bit of cleanup we could do in the matrix:include section, but I can handle that.

**Pull request merged**